### ISSUE TITLE
Benefits 2.5.1: split benefit and remove redundant one (for other SC)

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#747]
https://www.w3.org/WAI/WCAG21/Understanding/pointer-gestures.html#benefits

> Benefits
> - Users who cannot (accurately) perform complex pointer gestures will have alternative means for operating the content.
> - The availability of user interface elements provided as a fallback for complex gestures helps users who may often be unaware of the support for complex pointer gestures. This can be beneficial especially for users with cognitive or learning disabilities.

Benefit 1: Fine, covers the SC text but would expect "multipoint" and "path-based" to be separated into their own benefit for clarity

Benefit 2: this one may be a side effect but is not what the intent is about, keeping focus on the SC itself and not all side effects I prefer to see this one at another SC, not here. It may be confusing and have people give wrong comprehension of the SC. This benefit would be fine for the 2.2 "Indication of Gestures and Movement". The intent text is also not mentioning "user interface elements (in text)" as the mandatory alternative and the SC is not focussed at: "beneficial especially for users with cognitive or learning disabilities".

**Comment by patrickhlauke** [comment]
Agree about Benefit 2. It could still be kept around, but prefixed clearly as being a side effect, and not the goal of the SC

**Comment by patrickhlauke** [comment]
Made a slightly wider PR that covers this, as well as a few other niggles. See https://github.com/w3c/wcag/pull/1569

**Comment by jake-abma** [comment]
eager to see this on the agenda for discussion...
